Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -2377,9 +2378,9 @@ class CloneSerializer : public CloneBase {
 #if PLATFORM(COCOA)
     void write(const RetainPtr<CFDataRef>& data)
     {
-        uint32_t dataLength = CFDataGetLength(data.get());
-        write(dataLength);
-        write(CFDataGetBytePtr(data.get()), dataLength);
+        auto span = toSpan(data.get());
+        write(static_cast<uint32_t>(span.size()));
+        write(span);
```

Comment:
I'm not sure this trip through `span` makes this code better. Though, if `write()` had an overload that took a `span`, that might make it better.

Reply:
Oh, I guess it does, but only for a `span<const uint8_t>`. Maybe adding another of `span<const uint32_t>` would be too much.

Reply:
I don't understand the feedback here. write() does take a span and I am passing it a span. I merely constructed the span a little earlier to avoid calling `CFDataGetLength()` twice, once explicitly and a second time inside `toSpan()`.

Reply:
I was just pointing out that since a `span` knows its size, the serializer needing to accept a size, then pass the span (which knows the size) seems like extra effort. Especially if every SerializedScriptValue user with a span has to know to send the two values independently.